packages/gatsby/src/utils/webpack.config.js

```diff
@@ -227,6 +228,10 @@ module.exports = async (
             plugins.hotModuleReplacement(),
             plugins.noEmitOnErrors(),
             plugins.eslintGraphqlSchemaReload(),
+            // Don't use the default mini-css-extract-plugin setup as that
+            // breaks hmr.
+            new MiniCssExtractPlugin({ filename: `[name].css` }),
```

Comment:
the other webpack change — extract all css for adding to the `<head>` like we do in prod